tests/framework/extensions/clusters/import.go

```diff
@@ -137,7 +141,10 @@ func ImportCluster(client *rancher.Client, cluster *management.Cluster, rest *re
 						{
 							Name:    "kubectl",
 							Image:   "rancher/shell:v0.1.10",
-							Command: []string{"kubectl", "apply", "--filename=" + token.ManifestURL},
+							Command: []string{"/bin/sh", "-c"},
+							Args: []string{
+								fmt.Sprintf("wget -O %s --tries=10 --no-check-certificate %s; kubectl apply -f %s ;", applyFile, token.ManifestURL, applyFile),
```

Comment:
why do a format for the apply file?

Reply:
I changed I didn't realize there was a flag for `wget` to pipe out to stdout. How does this look now?